Webpack build throws mising dependency error

**Describe the bug**
When building an app using Webpack 4.35.0 to build an app using Vue 2.6.10 and Plyrue 2.0.4, Webpack throws a _Module not found_ error:

```
ERROR in ./node_modules/plyrue/dist/plyrue.js
Module not found: Error: Can't resolve 'vue-runtime-helpers/dist/normalize-component.js' in '/Users/julien/projects/ideation6/front/node_modules/plyrue/dist'
 @ ./node_modules/plyrue/dist/plyrue.js 5:40-98
 @ ./src/main.js
```

**To Reproduce**
1. Create a basic Vue project with a _App.vue_ component
2. Add Plyrue as a dependency: `npm i plyrue`
3. Register Plyrue as a plugin in Vue, using the instructions in the doc:
```
import App from './App.vue';
import Plyrue from 'plyrue';
import Vue from 'vue';

Vue.use(Plyrue);

new Vue({
  render: h => h(App),
}).$mount('#app');
```
4. Build the app using Webpack: `npx webpack --progress`
5. The Webpack build fails and outputs the error message

**Expected behavior**
The Webpack build completes successfully.

**Desktop:**
 - OS: MacOS 10.14.5
 - Browser: Firefox
 - Version: 68.0.1
 - Node 12.4.0
 - NPM 6.9.0

**Additional context**
The issue also appears when running the Webpack dev server.

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "lint-staged": "^9.2.1",
     "prettier": "^1.18.2",
     "rimraf": "^2.6.3",
-    "rollup-plugin-vue": "5.0.1",
+    "rollup-plugin-vue": "4.3.2",
     "tailwindcss": "^1.1.0",
     "vue": "^2.6.10",
     "vue-router": "^3.1.1",

